src/bundle/Resources/views/parts/menu/top_menu.html.twig

```diff
@@ -22,7 +22,7 @@
     {% set linkAttributes = item.linkAttributes|merge({
         'href': item.uri is not empty ? item.uri : '#' ~ item.name,
         'aria-controls': item.name,
-        'class': item.linkAttributes|default('') ~ (' nav-link' ~ (matcher.isAncestor(item, options.matchingDepth) ? ' ' ~ options.currentClass))|trim,
+        'class': ('nav-link' ~ (matcher.isAncestor(item, options.matchingDepth) ? ' ' ~ options.currentClass|trim)) ~  (item.linkAttributes.class is defined ?  ' ' ~ item.linkAttributes.class|trim),
```

Comment:
@arfaram Isn't `item.linkAttributes.class|default('')` => `item.linkAttributes.class|default('')` a sufficient fix?

Reply:
@webhdx also works but I need to add a blank before when it is defined.